peripherals/syscfg/syscfg_f301.yaml

```
@@ -0,0 +1,95 @@
+SYSCFG:
+  CFGR1:
+    I2C3_FMP:
+      Standard: [0, "FM+ mode is controlled by I2C_Pxx_FMP bits only"]
+      FMP: [1, "FM+ mode is enabled on all I2C3 pins selected through selection trhough IOPORT control registers AF selection bits"]
+    ENCODER_MODE:
+      NoRedirection: [0, "No redirection"]
+      # TODO: Naming?
+      MapTim2Tim15: [1, "TIM2 IC1 and TIM2 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+    TIM6_DAC1_DMA_RMP:
+      NoRemap: [0, "TIM6_UP and DAC_CH1 DMA requests mapped on DMA2 channel 3"]
+      Remap: [1, "TIM6_UP and DAC_CH1 DMA requests mapped on DMA1 channel 3"]
+    TIM1_ITR3_RMP:
+      NoRemap: [0, "No remap"]
+      Remap: [1, "TIM1_ITR3 = TIM17_OC"]
```

What about the naming convention in this case, should'nt it technically be:

```suggestion
      NotRemapped: [0, "No remap"]
      Remapped: [1, "TIM1_ITR3 = TIM17_OC"]
```

Just trying to figure out the naming convention here. I don't really mind, which to prefer, as they both convey the message similarly, though in this case, the flag is used for configuration (user doing it actively) as opposed to reading out the current state of the hardware (in a general sense). But it is still a read-write flag.

---

Good point, yes they should be `NotRemapped`/`Remapped` please.